src/main/java/courgette/runtime/CourgetteRunOptions.java

```diff
@@ -35,6 +35,11 @@ public boolean showTestOutput() {
         return SystemPropertyUtils.getBoolProperty(CourgetteSystemProperty.SHOW_TEST_OUTPUT, courgetteOptions.showTestOutput());
     }
 
+    @Override
+    public String reportTargetDir() {
+        return System.getProperty(CourgetteSystemProperty.REPORT_TARGET_DIR, courgetteOptions.reportTargetDir());
```

Comment:
Can you move this to the **SystemPropertyUtils** and ensure the provided value is not empty. If it is, default to courgetteOptions.reportTargetDir() 

If I were to provide this, it will be accepted even though it shouldn't!
````gradle
-Dcourgette.reportTargetDir = " "
````